usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -256,49 +243,19 @@ def test_cluster_import_valid(valid_session_credentials, valid_trusted_pool):
 """
 
 
-@pytest.mark.parametrize("node_ids,asserts", [
-    (["000000-0000-0000-0000-000000000"], {
-        "json": json.loads(
-            '{"errors": "Node 000000-0000-0000-0000-000000000 not found"}'),
-        "cookies": None,
-        "ok": False,
-        "reason": 'Unprocessable Entity',
-        "status": 422,
-        })])
+@pytest.mark.parametrize("cluster_id", [
```

Comment:
What is the error code here as API response? There should be some and this tescase should reflect that.

Reply:
API call with this cluster_id doesn 't return error but only job_id. Tendrl then starts processing the job and it fails during processing. There is no error in response only in job status.

Reply:
Ok, please add "failure" state to fixture, so it can be easily changed.